management/mailconfig.py

```diff
@@ -125,13 +139,46 @@ def get_mail_users_ex(env, with_archived=False):
 	users = []
 	active_accounts = set()
 	c = open_database(env)
-	c.execute('SELECT email, privileges FROM users')
-	for email, privileges in c.fetchall():
+	c.execute('SELECT email, privileges, quota FROM users')
+	for email, privileges, quota in c.fetchall():
 		active_accounts.add(email)
 
+		(user, domain) = email.split('@')
+		box_size = 0
+		box_count = 0
+		box_quota = 0
+		percent = ''
+		try:
+			dirsize_file = os.path.join(env['STORAGE_ROOT'], 'mail/mailboxes/%s/%s/maildirsize' % (domain, user))
+			with open(dirsize_file, 'r') as f:
+				box_quota = int(f.readline().split('S')[0])
+				for line in f.readlines():
+					(size, count) = line.split(' ')
+					box_size += int(size)
+					box_count += int(count)
+
+			try:
+				percent = (box_size / box_quota) * 100
+			except:
+				percent = 'Error'
+
+		except:
+			box_size = '?'
+			box_count = '?'
+			box_quota = '?'
+			percent = '?'
+
+		if quota == '0':
+			percent = ''
+
 		user = {
 			"email": email,
 			"privileges": parse_privs(privileges),
+            "quota": quota,
+			"box_quota": box_quota,
+			"box_size": sizeof_fmt(box_size) if box_size != '?' else box_size,
+			"percent": '%3.0f%%' % percent if type(percent) != str else percent,
+			"box_count": box_count,
```

Comment:
It's used in the management users.html template mostly.  I'm not sure how useful this is to have there, but maybe @stylnchris could shed some light on this for us?

Reply:
Number of messages can be removed. I do find it somewhat helpful but I do understand that its not part of the scope of adding quotas.   This would be "another feature" I suppose.

Reply:
[The counting of messages](https://github.com/mail-in-a-box/mailinabox/pull/2387#discussion_r1623525354) has been removed here: https://github.com/mail-in-a-box/mailinabox/pull/2387/commits/c91db7bc4f801e00fa59b87d296e424e8b5d5e41